### ISSUE TITLE
WSL2 iptables port forwarding from Distro port to Windows port host doesn't work

### ISSUE THREAD

**Issue by mdzhigarov** [microsoft/WSL#6083]
# Environment

```none
Windows build number: **[Version 10.0.18363.1082]**
Your Distribution version: **Release:        20.04**
Whether the issue is on WSL 2 and/or WSL 1: **WSL 2**
```

# Steps to reproduce

Start Apache http server on port 8282 on the Windows host. 

Log into WSL Ubuntu distro and execute the following set of commands:

`export WSL_WINDOWS_HOST=$(tail -1 /etc/resolv.conf | cut -d' ' -f2)`
```
export WSL_LOCAL_IP=`ip addr show eth0 | grep -oP '(?<=inet\s)\d+(\.\d+){3}'`
```

`sudo vim /etc/sysctl.conf `
... Uncomment net.ipv4.ip_forward = 1

`sudo sysctl -p`
`sudo sysctl --system`

`sudo iptables -t nat -A PREROUTING -p tcp --dport 8282 -j DNAT --to-destination $WSL_WINDOWS_HOST:8282                              `
`sudo iptables -t nat -A POSTROUTING -p tcp -d $WSL_WINDOWS_HOST --dport 8282 -j SNAT --to-source $WSL_LOCAL_IP`

`telnet $WSL_WINDOWS_HOST 8282` -> Connects as expected

`telnet $WSL_LOCAL_IP 8282` -> Fails to connect -  Unable to connect to remote host: Connection refused

#  Expected behavior

Since Apache is running on port 8282 on the Windows host and port forwarding rules were added to WSL, I expect that telnet telnet $WSL_LOCAL_IP 8282 actually works and connects to the Windows host.

# Actual behavior

I'm getting connection refused.

Basically, what I'm trying to achieve is to forward all traffic on port 8282 of the WSL distro to port 8282 to the Windows host

**Comment by NoMindToThink** [comment]
hi, how to solve it?

**Comment by XeonKHJ** [comment]
How does it work?
